specification/dimensions/tags.md

```diff
@@ -0,0 +1,62 @@
+# Tags
+
+Tags are the set of finalized User-Defined and/or Provider-Defined Tags assigned to Tag Sources.  Tags commonly add business context to billing data to identify and accurately allocate charges.
+
+If a Tag is subject to a set of prioritized rules for determining its value, it is finalized after a single value is chosen from a set of corresponding values.  Otherwise, a Tag and a finalized Tag are equal.
```

Comment:
I am not understanding what "finalized tag" means...   Needs more context please.

Reply:
Added a few options here.  Any of these clearer or not?

Option 1 (original):
```
If a Tag is subject to a set of prioritized rules for determining its value, it is finalized after a single value is chosen from a set of corresponding values.  Otherwise, a Tag and a finalized Tag are equal.
```

Option 2:
A Tag can either be static or dynamic.  If a Tag is static, its value is immutable.  If a Tag is dynamic, its value is determined by a set of predefined user or provider rules.  A finalized Tag is the final result of any static or dynamic Tag.

Option 3:
A Tag is typically unaltered after being attached to a Tag Source.  In special circumstances, predefined rules configured by a user or provider, can dynamically determine a Tag's value.  In any case, a finalized Tag is the Tag's key and resulting value.

Reply:
I like option #2

Reply:
@ljadvey any thoughts here?

Reply:
Yes, #2 I agree with as well!

Reply:
Ok, settling on Option 2 and will see how this lands with others.